website/src/content/api-reference/commercetools-frontend-cypress.mdx

```diff
@@ -34,22 +34,100 @@ module.exports = (on, config) => {
 };
 ```
 
+<Info>
+
+For Cypress version >= `10.0.0` the configuration changed. See [migration guide](https://docs.cypress.io/guides/references/migration-guide#Migrating-to-Cypress-version-10-0).
+
+The task should be defined in the `cypress.config.ts` file.
+
+```ts title="cypress.config.ts"
+import { defineConfig } from 'cypress';
+import { customApplicationConfig } from '@commercetools-frontend/cypress/task';
+
+export default defineConfig({
+  retries: 1,
+  video: false,
+  e2e: {
+    async setupNodeEvents(on, cypressConfig) {
+      on('task', {
+        customApplicationConfig,
+      });
+      return cypressConfig
+    },
+    baseUrl: 'http://localhost:3001',
+  },
+});
+```
+
+</Info>
+
 Extend the Cypress commands:
 
 ```js title="cypress/support/commands.js"
 import '@commercetools-frontend/cypress/add-commands';
 ```
 
-## Commands
+# Commands
+
+## cy.loginToMerchantCenter
+
+This command performs the user login to the Merchant Center. It automatically detects whether the application is running on localhost or on production and chooses the appropriate login mechanism.<br/>
+
+- When the application runs locally, the same mechanism used in the `cy.loginByOidc` is used.
+- When the application runs on production, a normal login flow is used where the user credentials are typed into the login form.
+
+### Usage
+
+```ts
+it('should render page 1', () => {
+  cy.loginToMerchantCenter({
+    entryPointUriPath: 'my-app',
+    initialRoute: `/${Cypress.env('PROJECT_KEY')}/my-app/page-1`,
+  });
+})
+it('should render page 2', () => {
+  cy.loginToMerchantCenter({ entryPointUriPath: 'my-app' });
+  cy.visit(`/${Cypress.env('PROJECT_KEY')}/my-app/page-2`)
+})
+```
+
+### Options
+
+Available options are:
+
+* `entryPointUriPath` (**required**): The application entry point URI path. This value is used to identify the correct application config.
+* `dotfiles` (optional): Pass a list of dotenv files that must be loaded when the `custom-application-config.json` is loaded (in case you are using environment placeholder). By default the following dotenv files are loaded: `.env` and `.env.local`. You can also define the values using paths relative to the application folder.
```

Comment:
```suggestion
* `dotfiles` (optional): A list of dotenv files to load when the `custom-application-config.json` is loaded (in case you're using an environment placeholder). By default the following dotenv files are loaded: `.env` and `.env.local`. You can also define the values using paths relative to the application folder.
```